### PR TITLE
Update glossary

### PR DESCRIPTION
**Pull request by zklaus**
PR Checklist:

- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [x] put any other relevant information below

This PR is a minor update to the glossary, with style homogenization and a new entry for CDN.

### CODE REVIEW

**Comment by jaimergp** [comment]
The `<a id ...` anchors were there to re-export the old anchors from the old website, but we don't need to follow that format everywhere. Actually I lean towards moving away from the `#term-` prefix if possible (and only keep the old ones around for backwards compatibility with external sites linking to us, while using the new ones internally).

**Comment by zklaus** [comment]
> The `<a id ...` anchors were there to re-export the old anchors from the old website, but we don't need to follow that format everywhere. Actually I lean towards moving away from the `#term-` prefix if possible (and only keep the old ones around for backwards compatibility with external sites linking to us, while using the new ones internally).

Makes sense to me. That means keeping the `<a ...` tags, but removing the `{#term-...}` tags and updating all links within the repo that refer to `glossary[.md]#term-` to remove the `term-` part, correct?

**Comment by jaimergp** [comment]
Correct.

**Comment by zklaus** [comment]
I think this is ready to go.

File: docs/glossary.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
----
-title: 'Glossary'
```
Comment (jaimergp) [inline docs/glossary.md:2]: I think we need this line back (or push the `# Glossary` header to the absolute first line before the anchors). Otherwise the sidebar renders the title in lowercase.

File: docs/glossary.md [hunk 2]
```diff
@@ -11,74 +6,116 @@ sidebar_position: 27
 
 # Glossary
 
+List of frequently used terms and acronyms.
+
 <a id="term-ABI"></a>
 
 ## ABI
 
-**A**pplication **B**inary **I**nterface. ABI is a document that comprehensively defines the binary system interface between applications and the operating system on which they run. [Learn More at Wikipedia](https://en.wikipedia.org/wiki/Application_binary_interface) or [pypackaging-native](https://pypackaging-native.github.io/background/binary_interface).
```
Comment (jaimergp) [inline docs/glossary.md:18]: Why did you remove the `pypackaging` reference?

File: docs/glossary.md [hunk 3]
```diff
@@ -11,74 +6,116 @@ sidebar_position: 27
...
+Content Delivery Network. CDNs are geographically distributed networks of servers that mirror the contents of a primary source. Having multiple servers offering the same content increases performance (reduced latency, higher download speeds) and availability.
+
+[Learn more](https://en.wikipedia.org/wiki/Content_delivery_network).
 
 <a id="term-CDT"></a>
 
 ## CDT
 
-**C**ore **D**ependency **T**ree. Core Dependency Tree packages take care of the dependencies which are so close to the system that they are not packaged with conda-forge. A CDT package consists of repackaged CentOS binaries from the appropriate version, either 6 or 7 depending on user choice and platform. [Learn more](maintainer/knowledge_base.md#cdt-packages).
+Core Dependency Tree. Core Dependency Tree packages take care of the dependencies that are so close to the system that they are not packaged with conda-forge. A CDT package consists of repackaged CentOS binaries from the appropriate version, either 6 or 7 depending on user choice and platform.
```
Comment (jaimergp) [inline docs/glossary.md:31]: This part will get out of date soon. I think it's better to have this last sentence present an example.

> For example, in conda-forge, we have used repackaged CentOS 6 or 7 for some time.

File: docs/glossary.md [hunk 4]
```diff
@@ -11,74 +6,116 @@ sidebar_position: 27
...
 
-<a id="term-CFEP"></a>
+[Learn More](https://en.wikipedia.org/wiki/Continuous_integration).
 
-## CFEP
+<a id="term-CLI"></a>
+
+## CLI
+
+Command Line Interface. A program in which commands are entered as text, one line at a time, for a computer to execute. This is done in the Miniforge Prompt in Windows, and in a terminal in macOS and Linux. `conda` is executed in a CLI.
```
Comment (jaimergp) [inline docs/glossary.md:55]: I would rephrase the last two sentences to make sure people understand they are concrete examples.

File: docs/glossary.md [hunk 5]
```diff
@@ -11,74 +6,116 @@ sidebar_position: 27
...
+[Learn More](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/channels.html).
 
 <a id="term-Conda-package"></a>
 
 ## Conda package
 
-A conda package is a `.tar.bz2` or `.conda` archive that contains libraries, executable programs, data files and other components, as well as metadata under the `info/` directory. Its contents are unpacked in the installation prefix. [Learn More](<https://en.wikipedia.org/wiki/Conda_(package_manager)>).
+A conda package is a `.tar.bz2` or `.conda` archive that contains libraries, executable programs, data files and other components, as well as metadata under the `info/` directory. Its contents are unpacked in the installation prefix.
+
+[Learn More](<https://en.wikipedia.org/wiki/Conda_(package_manager)>).
```
Comment (jaimergp) [inline docs/glossary.md:73]:
```suggestion
[Learn More](https://en.wikipedia.org/wiki/Conda_(package_manager)).
```

File: docs/glossary.md (hunk 2 above)
Comment (zklaus) [reply docs/glossary.md:18]: Snafu. It's back now.

File: docs/glossary.md (hunk 4 above)
Comment (zklaus) [reply docs/glossary.md:59]: Hm. To be honest, I am not thrilled about the entire explanation. What do we mean here with CLI? Imho, it can be either "the program that takes the command-line and launches the desired program", i.e. power shell, bash, etc.; or it can be the interface offered by programs such as `conda` including parameters etc.

When I say CLI, I usually mean the latter. The explanation here starts out with the former, but than kind of mangles the two; imho, the linked explanation on wikipedia refers to the second meaning.

Comment (zklaus) [reply docs/glossary.md:59]: So I would most like to rewrite with the second meaning. Something along the lines of
```suggestion

## CLI

Command Line Interface. A textual interface of a program that allows the user to control the program with a single command line, which is entered in a command-line environment, such as the Miniforge Prompt on Windows, or a terminal on MacOS and Linux. An example for a program offering a CLI is `conda`.
```

**Comment by zklaus** [comment]
*sigh* link checker failure due to ci network problems. @conda-forge-admin, please restart ci

**Comment by jaimergp** [comment]
I don't think that works in this repo (but you can always close and reopen on your own). I issued a CI rerun, let's see.

**Comment by jaimergp** [comment]
Thanks!
